src/components/ForumCard.js

```diff
@@ -0,0 +1,262 @@
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    Forums
+                </Typography>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+                {forums.map((event) => (
+                    <Card style={{ width: '1000px', marginBottom: '20px' }} key={event.id}>
+                        <CardContent>
+                            <Link to={`/forum/${event.id}`} target="_blank">
+                                <Typography variant="h5" component="div">
+                                    {event.forumTitle}<br />
+                                </Typography>
+                            </Link>
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                Posted on {new Date(event.dateTime).toLocaleDateString()} by {event.creatorName}<br />
+                            </Typography>
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                <br />{event.description}<br />
+                            </Typography>
+                            <form onSubmit={handleApiAddComment}>
+                                <TextField
+                                    style={{ mb: 1.5, width: '350px' }}
+                                    label="Add a comment"
+                                    value={newComment}
+                                    onChange={handleNewComment}
+                                />
+                                <Button type="submit" style={{ color: "white", backgroundColor: "seagreen", mb: 1.5 }}>Submit</Button>
+                            </form>
+                            <Typography sx={{ mb: 1.5 }} variant="h6" color="text.secondary">
+                                <br />Comments:<br />
+                            </Typography>
+                            {comments.map((comment) => (
+                                <Card style={{ width: '500px', marginBottom: '20px' }} key={comment.id}>
+                                    <CardContent>
+                                        <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                            {comment.comment}<br />
+                                        </Typography>
+                                        <Typography style={{ mb: 1.5, fontSize: "12px" }} color="text.secondary">
+                                            <strong> By: </strong> {comment.firstName} {comment.lastName}<br />
```

Comment:
correct order should be:
name
comment
date

Reply:
thoughts on adding program and year? so something like
Safwaan Chowdhury
3A Management Engineering
some comment
date

Reply:
might run into some complications if the user doesn't have a user profile created... but in that case use conditional rendering to not display

doesn't need to be implemented but nice to have type beat

Reply:
Yeah, it's a great idea!! However, for consistency, I think we should avoid it till we can find a way that people always add their year/program first. 

Cause as someone viewing the comments: 

Safwaan Chowdhury
3A Management Engineering
some comment
date

VS. 

Safwaan Chowdhury
some comment
date

Can get a bit confused!